chef/cookbooks/nova_dashboard/templates/default/local_settings.py.erb

```diff
@@ -290,16 +290,16 @@ TIME_ZONE = "<%= @timezone %>"
 # target installation.
 
 # Path to directory containing policy.json files
-#POLICY_FILES_PATH = os.path.join(ROOT_PATH, "conf")
+POLICY_FILES_PATH = os.path.join(ROOT_PATH, '<%= @policy_file_path =>')
 # Map of local copy of service policy files
-#POLICY_FILES = {
-#    'identity': 'keystone_policy.json',
-#    'compute': 'nova_policy.json',
-#    'volume': 'cinder_policy.json',
-#    'image': 'glance_policy.json',
-#    'orchestration': 'heat_policy.json',
-#    'network': 'neutron_policy.json',
-#}
+POLICY_FILES = {
+    'identity': '<%= @policy_file[:identity] =>',
+    'compute': '<%= @policy_file[:compute] =>',
+    'volume': '<%= @policy_file[:volume] =>',
+    'image': '<%= @policy_file[:image] =>',
+    'orchestration': '<%= @policy_file[:orchestration] =>',
+    'network': '<%= @policy_file[:network] =>',
+}
```

Comment:
Hm, sorry. It seems I have completely missunderstood the meaning of the ```POLICY_FILES_PATH``` setting. My previous comment was nonsense :(